ROADMAP.md

```diff
@@ -0,0 +1,154 @@
+## Vision
+
+> Amethyst aims to be a modular, parallel and data-driven game engine
+
+**Modularity** The game engine should be easily extendable to fit the needs of the game developers for their game idea. Therefore we expose an entity-component system, which allows to define new components and processors without needing to change the game engine code. This should emphasize the 'game engine as library' aspect.
```

Comment:
I was talking about picking which language is used by default, it doesn't seem like we'd want to implement the actual scripts themselves by default.